### PR TITLE
Updates to csproj for RTM tooling

### PR DESCRIPTION
**Pull request by GuardRex**
@Rick-Anderson @tdykstra 

**Priority Update** for RTM tooling. https://github.com/dotnet/sdk/issues/396 is fixed.

Would you call in [@]vijayrkn or [@]natemcmaster to review?
Testing here is good, but I'd :heart: a dev team :eye: on this.

Shout out to @dasmulli 🌞 ... a constant source of help and inspiration.

### CODE REVIEW

File: aspnetcore/fundamentals/url-rewriting/sample/URLRewritingSample.csproj [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
+    <RuntimeFrameworkVersion>1.1.0</RuntimeFrameworkVersion>
```
Comment (dasMulli) [inline aspnetcore/fundamentals/url-rewriting/sample/URLRewritingSample.csproj]: That shouldn't be necessary for RTM.
It's possible to work around the packages missing from NuGet by adding https://dotnet.myget.org/F/dotnet-core/api/v3/index.json to a local NuGet.Config file

Comment (GuardRex) [inline aspnetcore/fundamentals/url-rewriting/sample/URLRewritingSample.csproj]: Removed ... thanks.

**Comment by Rick-Anderson** [comment]
@GuardRex do you really  _wants to merge 31 commits_

**Comment by GuardRex** [comment]
darn it! ... I didn't branch that correctly.

**Comment by GuardRex** [comment]
@Rick-Anderson I'll get back to you on this tonight. I'm on a `dotnet new` update thing, but I'll get this fixed tonight.

**Comment by tdykstra** [comment]
csproj branch was rebased on master -- can you change this to go into master?

**Comment by GuardRex** [comment]
@tdykstra I tried to PR from my csproj branch to master, but it's showing those 31 commits in the PR. Is that what you want to do here?

Next, I tried to rebase my csproj and it showed conflicts. I didn't want to risk a big git :boom:, so I aborted that.

If you want, I can just wait until you merge all of the csproj changes and get a quick PR in to master to fix these files. Otherwise, I'll need some guidance. The [@]pinpoint training course I took didn't cover advanced scenarios! lol

**Comment by tdykstra** [comment]
> wait until you merge all of the csproj changes and get a quick PR in to master to fix these files

We have instructions for bringing in the rebased csproj and rebasing your local branch on it, but the proposal to redo on master sounds easier.

**Comment by GuardRex** [comment]
Cool. Let's do that. I don't want to create a big mess on the first day! lol

**Pull request closed without merging**